src/Io/ChunkedDecoder.php

```diff
@@ -122,12 +122,12 @@ public function handleData($data)
                     }
                 }
 
-                $this->chunkSize = \hexdec($hexValue);
-                if (\dechex($this->chunkSize) !== $hexValue) {
+                if (\ctype_xdigit($hexValue) === false || \dechex(\hexdec($hexValue)) !== $hexValue) {
```

Comment:
Changes LGTM :+1:

Note that this requires `ext-ctype` which could potentially be disabled explicitly. As an alternative we could also keep the previous logic and suppress errors for PHP 7.4 support. What do you think about this?

Reply:
Was more thinking along the lines of using: https://github.com/symfony/polyfill-ctype

Reply:
Alternatively I can do a PHP version check and only run it on PHP 7.4 and above and do some extra trickery so we can seamlessly do this

Reply:
It's funny that there's no changelog about the deprecation notice on the function page.

Reply:
Oh well <https://bugs.php.net/bug.php?id=79103>

Reply:
lol doh I assumed I missed it :joy: